src/main/java/ch/njol/skript/expressions/ExprItemWithLore.java

```diff
@@ -62,10 +64,13 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean kleenean
 
 	@Override
 	protected ItemType[] get(Event e, ItemType[] source) {
-		String[] lore = this.lore.getArray(e);
+		List<String> lore = Arrays.stream(this.lore.getArray(e))
```

Comment:
stream looks nice, but you should be able to replace the Arrays#stream call with `this.lore.stream(e)`

Reply:
Thanks. I didn't know about such a nice method!